developerportal/templates/admin/login.html

```diff
@@ -0,0 +1,42 @@
+{% extends "admin/login.html" %}
+{% load i18n app_tags %}
+
+{% block content %}
+
+  {% use_conventional_auth as conventional_auth_enabled %}
+
+  {% if conventional_auth_enabled %}
+
+    {{block.super}}
+
+  {% else %}
+
+    {% if form.errors and not form.non_field_errors %}
+    <p class="errornote">
+    {% if form.errors.items|length == 1 %}{% trans "Please correct the error below." %}{% else %}{% trans "Please correct the errors below." %}{% endif %}
+    </p>
+    {% endif %}
+
+    {% if form.non_field_errors %}
+    {% for error in form.non_field_errors %}
+    <p class="errornote">
+        {{ error }}
+    </p>
+    {% endfor %}
+    {% endif %}
+
+    <p class="module">
+      <a class="button" href="{% url 'oidc_authentication_init' %}">
+        {% trans 'Sign in with Mozilla SSO' %}
+      </a>
+    </p>
+    <p>
+      Note that after sign-in, you will be sent back to the CMS admin. Please re-access {% url 'admin:index' %} manually.
```

Comment:
Not a super- clean way, because Django only takes a string (a path or URL name) for `LOGIN_REDIRECT_URL`. I _could_ add a custom view to handle the redirect and send people the right way, but given the very low likelihood of authorised people wanting the Django Admin itself (rather than just Wagtail's Admin), I thought a bit of help text + human brainpower will bridge the gap for now. We can revisit this post-alpha if it annoys people.